test/Tizen.NUI.WidgetViewTest/4.WidgetComponentTest/Tizen.NUI.WidgetViewTest/SimpleWidgetViewApp.cs

```diff
@@ -0,0 +1,101 @@
+﻿/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd.
```

Comment:
```suggestion
 * Copyright (c) 2024 Samsung Electronics Co., Ltd.
```

Reply:
Thank you for your review. I'll fix it quickly 😊